### ISSUE TITLE
When should we prepend ◌?

### ISSUE THREAD

**Issue by AlexDaniel** [perl6/whateverable#167]
See this [TODO](https://github.com/perl6/whateverable/blob/f5459ff7c93caa6c0724445f9d41846eee255939/Unicodable.p6#L82).

**Comment by AlexDaniel** [comment]
Done. See this discussion: https://irclog.perlgeek.de/perl6-dev/2017-07-18#i_14890127

**Comment by AlexDaniel** [comment]
Some say that NBSP does not render very well. Maybe we should use other character. Any feedback?

**Comment by AlexDaniel** [comment]
For example, irclog currently does not render combiners at all, so that's not an indicator.

**Comment by AlexDaniel** [comment]
FWIW, I remember that there was some proposed character exactly for that purpose. @samcv in the log above said “INVISIBLE CHARACTER”, but I can't find such thing right now. Not even sure if it made its way into the standard.

**Comment by samcv** [comment]
Yeah that never made it in. You are supposed to use NBSP. Have you tried possibly making multiple NBSP and see if that renders any better than only one?
